debian/make-boot-image.install

```diff
@@ -0,0 +1 @@
+tools/make-boot-image usr/share/rpiboot/tools/
```

Comment:
I'm in favour of prompting to /usr/bin. Left in this location, it's not going to be of huge use - unless we expect people to `dpkg -L` the package after install...

Reply:
Agree